kustomize/components/pgupgrade/quay-pg.statefulset.patch.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
```

Comment:
The original issue seems to be caused by the old and the new db pods being matched by the same service, making the requests against the db go to either or randomly. Does switching to stateful sets fix this issue?

Reply:
Yes, since their will be only one underlying pod attached to the service